bin/commands/channels.rb

```diff
@@ -53,128 +20,4 @@
       puts JSON.dump($client.channels_id(options))
     end
   end
-
-  g.desc 'Gets information about a channel.'
-  g.long_desc %( Gets information about a channel. )
-  g.command 'info' do |c|
-    c.flag 'channel', desc: 'Channel to get info on.'
-    c.flag 'include_locale', desc: 'Set this to true to receive the locale for this channel. Defaults to false.'
-    c.action do |_global_options, options, _args|
-      puts JSON.dump($client.channels_info(options))
-    end
-  end
-
-  g.desc 'Invites a user to a channel.'
-  g.long_desc %( Invites a user to a channel. )
-  g.command 'invite' do |c|
-    c.flag 'channel', desc: 'Channel to invite user to.'
-    c.flag 'user', desc: 'User to invite to channel.'
-    c.action do |_global_options, options, _args|
-      puts JSON.dump($client.channels_invite(options))
-    end
-  end
-
-  g.desc 'Joins a channel, creating it if needed.'
-  g.long_desc %( Joins a channel, creating it if needed. )
-  g.command 'join' do |c|
-    c.flag 'name', desc: 'Name of channel to join.'
-    c.flag 'team_id', desc: 'encoded team id to list channels in, required if org token is used.'
-    c.flag 'validate', desc: 'Whether to return errors on invalid channel name instead of modifying it to meet the specified criteria.'
-    c.action do |_global_options, options, _args|
-      puts JSON.dump($client.channels_join(options))
```

Comment:
I have been using `channels_join`. What else should I use know since this one is depricated?

Reply:
@DariuszMusielak I believe https://api.slack.com/methods/conversations.join is what you want to use (or `conversations_join` in the client). I can't find reference to any previous `channels.join` method in the latest API docs, but other `channels.*` methods have all been deprecated in favor of the `conversations` namespace.